arm/Microsoft.DocumentDB/databaseAccounts/.parameters/plain.parameters.json

```
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "sxx-az-cdb-x-003"
+            "value": "sxx-az-cdb-pln-001"
```

```suggestion
            "value": "sxx-az-cdb-plain-001"
```

or rename the parameter file to pln.parameters.json